frontend/packages/console-shared/src/components/formik-fields/field-types.ts

```diff
@@ -44,6 +44,7 @@ export interface DropdownFieldProps extends FieldProps {
   title?: React.ReactNode;
   fullWidth?: boolean;
   disabled?: boolean;
+  autoComplete?: boolean;
```

Comment:
There are several instances in the console where an `autocompleteFilter` other than `fuzzy` is being used. I suggest we keep the `DropdownField` generic and accept an auto complete filter function.

```suggestion
  autocompleteFilter?: (text: string, item: object, key: string) => boolean;
```

Reply:
updated